src/skills-builder/skills-builder-header/SkillsBuilderHeader.jsx

```diff
@@ -1,25 +1,39 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { useIntl } from '@edx/frontend-platform/i18n';
 import edXLogo from '../images/edX-logo.svg';
 import messages from './messages';
 
-const SkillsBuilderHeader = () => {
+const SkillsBuilderHeader = ({ isMedium }) => {
   const { formatMessage } = useIntl();
 
+  if (isMedium) {
+    return (
+      <div className="d-flex">
+        <h3 className="mb-0 text-white">
+          {formatMessage(messages.skillsBuilderHeaderTitleIsMedium)}
+        </h3>
+      </div>
+    );
+  }
   return (
     <div className="d-flex">
       <img src={edXLogo} alt="edx-logo" className="mt-2 h-50" />
       <div className="ml-5 vertical-line" />
       <div className="w-100 ml-5">
-        <h1 className="h1 text-warning-300">
+        <h1 className="text-warning-300">
           {formatMessage(messages.skillsBuilderHeaderTitle)}
         </h1>
-        <p className="h2 text-white">
+        <h2 className="text-white">
           {formatMessage(messages.skillsBuilderHeaderSubheading)}
-        </p>
+        </h2>
```

Comment:
Similarly, related to my above comment, should this really be an h2? Previously, it was just text _styled_ like an H2, which (I thought) I had been told was preferable than defining header levels directly.